services/ingestion/event_subscriber.go

```diff
@@ -24,33 +24,46 @@ type EventSubscriber interface {
 	//
 	// The BlockEvents type will contain an optional error in case
 	// the error happens, the consumer of the chanel should handle it.
-	Subscribe(ctx context.Context, height uint64) <-chan models.BlockEvents
+	Subscribe(ctx context.Context) <-chan models.BlockEvents
 }
 
-var _ EventSubscriber = &RPCSubscriber{}
+var _ EventSubscriber = &RPCEventSubscriber{}
 
-type RPCSubscriber struct {
-	client            *requester.CrossSporkClient
-	chain             flowGo.ChainID
-	heartbeatInterval uint64
-	logger            zerolog.Logger
+type RPCEventSubscriberConfig struct {
+	HeartbeatInterval uint64
```

Comment:
Nice :ok_hand: 
Maybe we should also add `height` and `chain` here as well? What do you think?